bin/clever.js

```diff
@@ -501,6 +500,13 @@ function run () {
     options: [opts.alias],
   }, open);
 
+    // CONSOLE COMMAND
+    const console = lazyRequireFunctionWithApi('../src/commands/console.js');
```

Comment:
I'm afraid this naming could confuse people with the already existing global variable `console`.

* Could you rename this?
* Maybe something like `consoleModule` or `consoleFn`